src/web/core/src/main/java/org/geoserver/web/data/layer/CascadedWFSStoredQueryAbstractPage.html

```diff
@@ -18,7 +18,7 @@
 
 <wicket:fragment wicket:id="newPage">
   <div>
-    <wicket:message key="chooseNativeName">Layer name </wicket:message> <input type="text" wicket:id="nativeName"></input>
+    <wicket:message key="chooseNativeName">Layer name </wicket:message> <input type="text" wicket:id="nativeName">
```

Comment:
Not clear on what you mean by "training".
Yes, it should be <input /> but was just trying to focus on getting this ticket done and most browsers are ok with just <input>.